### PR TITLE
fix(telephony.group.consumption.pie-chart): improve responsiveness

### PR DESCRIPTION
**Pull request by Heiiji**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| License          | BSD 3-Clause

## Description

Apply a responsive comportment on telecom graph (small screen).

Before : 
before

After : 
Capture d’écran 2020-10-20 175534

### CODE REVIEW

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.directive.js [hunk 1]
```diff
@@ -98,6 +104,10 @@ angular
                 currentArc.classed("clicked", !clicked);
             }); */
 
+        angular.element($window).bind('resize', function() {
```
Comment (frenautvh) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.directive.js:107]: you should unbind when scope is destroyed

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="pie__legend">
+<div class="{{pieLegendClass()}}">
```
Comment (frenautvh) [inline packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.html:1]: maybe you can use ng-class

Comment (Heiiji) [reply packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.html:1]: I've try with ng-class but i've fail. What i put in ng-class isn't interpreted and i have the function name on class name place :/
i have try : 
 - data-ng-class="pieLegendClass()"
 - data-ng-class="toggle ? 'pie__legend' : 'pie__legend__subtitle'"

this 2 isn't interpreted and came on DOM.

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.directive.js (hunk 1 above)
Comment (Heiiji) [reply packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.directive.js]: I have fixe that on last commit, can you re-check please ? :)

File: packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.html (hunk 2 above)
Comment (JeremyDec) [reply packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.html:1]: @Heiiji what about that:
pie__legend__subtitle' : 'pie__legend
```suggestion
<div class="{ 'pie__legend': toggle, 'pie__legend__subtitle': !toggle }">
```

Comment (Heiiji) [reply packages/manager/apps/telecom/src/components/telecom/telephony/group/consumption/pie-chart/telephony-group-consumption-pie-chart.html]: yeah, thanks :) i've push this modif and it's work. I'm not very familliar with angular DOM logic for moment. ^^

**Review by JeremyDec — DISMISSED** [review]

**Review by Jisay — APPROVED** [review]

**Comment by Heiiji** [comment]
Got 2 approvals before applying code changes => i've remove dot on <ul> list

**Review by antleblanc — APPROVED** [review]

**Pull request merged**